static/js/biomedical/disease/page.tsx

```diff
@@ -65,6 +68,8 @@ export class Page extends React.Component<PagePropType, PageStateType> {
     );
   }
   render(): JSX.Element {
+    const diseaseName = getDiseaseCommonName(this.state.data);
+    const diseaseLink = `${GRAPH_BROWSER_REDIRECT}${this.props.dcid}`;
```

Comment:
Would be less error-prone if instead of a const, graph_browser_redirect was a function that returns a link given a DCID.

Reply:
Great suggestion! Added a function `getEntityLink()` in `bio_charts_utils` as it will be a shared function across diseases and proteins